### PR TITLE
better error message, optionally exit if no file

### PR DESCRIPTION
**Pull request by chriselion**
### Proposed change(s)
Computing inference rewards on daily training broke, because the model files changed directory. This improves the error logging to the inference script, and adds an option to exit if the file couldn't be loaded.

### Types of change(s)

- [x] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class ModelOverrider : MonoBehaviour
     {
         const string k_CommandLineModelOverrideFlag = "--mlagents-override-model";
         const string k_CommandLineQuitAfterEpisodesFlag = "--mlagents-quit-after-episodes";
+        const string k_CommandLineQuitOnLoadFailure = "--mlagents-quit-on-load-failure";
```
Comment (chriselion) [inline Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs:25]: @xiaomaogy you'd want to add this to the commandline args

File: Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs [hunk 2]
```diff
@@ -49,18 +52,22 @@ void GetAssetPathFromCommandLine()
...
                 if (args[i] == k_CommandLineModelOverrideFlag && i < args.Length-2)
                 {
                     var key = args[i + 1].Trim();
                     var value = args[i + 2].Trim();
                     m_BehaviorNameOverrides[key] = value;
                 }
-                else if (args[i] == k_CommandLineQuitAfterEpisodesFlag)
+                else if (args[i] == k_CommandLineQuitAfterEpisodesFlag && i < args.Length-1)
                 {
                     Int32.TryParse(args[i + 1], out maxEpisodes);
```
Comment (surfnerd) [inline Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs:65]: could you increment `i` here?

Comment (chriselion) [reply Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs:65]: Could, but I think this is clearer.

File: Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs (hunk 1 above)
Comment (xiaomaogy) [reply Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs:25]: This is perfect.

**Pull request merged**